adventure.py

```diff
@@ -26,10 +28,24 @@
 else:
     with_colors = False
 
-if platform.system() != "Windows":
+if platform.system() != "Windows": # compatibility fixes
     clear_command = "clear"
 else:
     clear_command = "cls"
+
+with_custom = input("Generate map or use custom?") # asking user to either generate or use custom map
```

Comment:
It would make more sense to have something like,
`if "generate" in with_custom.lower():`
because then if the user types some thing like "generate map" then "generate" is in the input, so it evaluates to `True`.

Reply:
it doesn't really matter that much, and I don't feel like rewriting it. Users aren't that stupid.

Reply:
@amistele I think it might be better if in a later commit we implement what @K-Mistele is suggesting... or at least wrap it in try except while catch loop